MANIFEST.in

```diff
@@ -21,6 +21,7 @@ exclude tox.ini
 exclude noxfile.py
 
 recursive-include src/pip/_vendor *.pem
+recursive-include src/pip/_vendor py.typed
```

Comment:
Hmm, I don't think there's much point in doing this -- we're shadowing the relevant modules through `.pyi` files in the vendor directory.